src/renderer/stores/chatList.js

```diff
@@ -11,15 +11,19 @@ const chatListStore = new Store(defaultState)
 // value for text1Meaning
 const DC_TEXT1_DRAFT = 1
 
+function sortChatList (first, second) {
+  return first.summary.timestamp > second.summary.timestamp ? -1 : 1
```

Comment:
I think that sorting the chatlist isn't needed, instead we should have another array which just holds the chatIds in the correct order. This we can just refetch with one call. But I will try to implement that on top of this, first lets get this bug fixed.